ably/docs
# [FEA-885] change term to state persistence

Explain why this change is being made.
----
In the Channel rules page change the term “last message persisted” to state persistence

Explain how this is accomplished.
----
copy was changed


How do you manually test this?
----
Navigate to http://ably-docs-pr-1042.herokuapp.com/general/channel-rules-namespaces/

copy of the second element of the list should start with  'State persistence'

Merge/Deploy Checklist
----


Frontend Checklist
----

- [ ] Have you compared with the initial design / fit our [brand guidelines](https://brand.ably.com/)?
